package.json

```diff
@@ -1,8 +1,40 @@
 {
   "name": "bats",
-  "version": "v1.0.1",
+  "version": "0.4.0",
   "description": "Bash Automated Testing System",
   "homepage": "https://github.com/bats-core/bats-core#readme",
-  "repository": "github:bats-core/bats-core",
-  "bin": "./bin/bats"
+  "license": "MIT",
+  "author": "Sam Stephenson <sstephenson@gmail.com> (http://sstephenson.us/)",
```

Comment:
Though Sam's the original author, of course, I wonder if it wouldn't be more accurate to list https://github.com/bats-core as the author and Sam as a contributor.

Reply:
Eh, the definitions I typically adhere to:

Author: The person/s or organization that created the project
Owner: The person/s who has administrative ownership over the organization or repository
Maintainers: Contributors who are responsible for driving the vision and managing the organizational aspects of the project.
Contributors: Everyone who has contributed something back to the project.

https://opensource.guide/how-to-contribute/#anatomy-of-an-open-source-project

Obviously, there are many (conflicting) definitions and there's nothing that states github's version as authoritative. However, by switching to AUTHORS file for contributors, that _kinda_ muddies the distinction (which could be seen as a good thing?)

I'd prefer to keep Sam as author in package.json, and move contributors to AUTHORS file. google seems to distinguish AUTHORS file from CONTRIBUTORS file: https://opensource.google.com/docs/releasing/authors/

I wouldn't mind re-addressing this if/when bats-core gets the official nod as maintained successor to bats.

Reply:
As far as an official nod goes, it'd be lovely if that would happen, and I'd welcome and celebrate it, as that's the order of things I'd prefer (short of being able to contribute to the original project, obviating the need for a fork)—but I've stopped holding my breath.

Reply:
Same. But until that happens and/or we significantly change the code, I'd prefer to just maintain status quo with him as author?

Reply:
It's arguable we have, but it's not a blocker, just something about which I was curious to hear others' thoughts.